src/views/admin/PersonImportView.vue

```diff
@@ -0,0 +1,583 @@
+                data-testid="download-file-button"
+              >
+                {{ $t('admin.import.downloadUserdata') }}
+              </v-btn>
+            </v-col>
+          </v-row>
+        </v-container>
+      </template>
+
+      <!-- Import loading template -->
+      <template v-if="importStore.importIsLoading && !importStore.errorCode">
+        <v-container>
+          <v-row justify="center">
+            <v-col cols="auto">
+              <v-progress-circular
+                data-testid="import-progress-spinner"
+                indeterminate
+                size="64"
+              ></v-progress-circular>
+            </v-col>
+          </v-row>
+        </v-container>
+      </template>
+
+      <!-- Upload form -->
+      <FormWrapper
+        v-if="!importStore.uploadResponse?.isValid && !importStore.errorCode"
+        :createButtonLabel="$t('admin.import.uploadFile')"
+        :discardButtonLabel="$t('nav.backToList')"
+        id="person-import-form"
+        :onDiscard="navigateToPersonTable"
+        :onSubmit="onSubmit"
+      >
+        <!-- Schulauswahl -->
+        <FormRow
+          :errorLabel="selectedSchuleProps['error']"
+          :isRequired="true"
+          labelForId="schule-select"
+          :label="$t('admin.schule.schule')"
+        >
+          <v-autocomplete
+            autocomplete="off"
+            clearable
+            @click:clear="clearSelectedSchule"
+            data-testid="schule-select"
+            density="compact"
+            id="schule-select"
+            ref="schule-select"
+            :items="schulen"
+            item-value="value"
+            item-text="title"
+            :no-data-text="$t('noDataFound')"
+            :placeholder="$t('admin.schule.selectSchule')"
+            required="true"
+            variant="outlined"
+            v-bind="selectedSchuleProps"
+            v-model="selectedSchule"
+            v-model:search="searchInputSchule"
+          ></v-autocomplete>
+        </FormRow>
+
+        <!-- Rollenauswahl (currently limited to SuS) -->
+        <FormRow
+          :errorLabel="selectedRolleProps['error']"
+          labelForId="rolle-select"
+          :isRequired="true"
+          :label="$t('admin.rolle.rolle')"
+        >
+          <v-autocomplete
+            autocomplete="off"
+            clearable
+            @click:clear="clearSelectedRolle"
+            data-testid="rolle-select"
+            density="compact"
+            id="rolle-select"
+            ref="rolle-select"
+            :items="lernRollen"
+            item-value="value"
+            item-text="title"
+            :no-data-text="$t('noDataFound')"
+            :placeholder="$t('admin.rolle.selectRolle')"
+            required="true"
+            variant="outlined"
+            v-bind="selectedRolleProps"
+            v-model="selectedRolle"
+          ></v-autocomplete>
+        </FormRow>
+
+        <!-- File Upload -->
+        <FormRow
+          :errorLabel="''"
```

Comment:
```suggestion
          :errorLabel="selectedFilesProps['error']"
```
Geht das so? damit der Label auch auf Rot gesetzt wird on error?